controllers/machine_controller.go

```diff
@@ -277,7 +277,7 @@ func (r *MachineReconciler) reconcileDelete(ctx context.Context, cluster *cluste
 	log := ctrl.LoggerFrom(ctx, "cluster", cluster.Name)
 
 	err := r.isDeleteNodeAllowed(ctx, cluster, m)
-	isDeleteNodeAllowed := err == nil //nolint:ifshort
+	isDeleteNodeAllowed := err == nil // nolint:ifshort
```

Comment:
The previous should be correct?

Reply:
It should. (we don't have a space in other cases).

I personally get errors that the nolint is not required at all at least in 1/10 of my `make lint` runs.

I'm not sure what the problem is but there seems to be some non-deterministic bug in the linter imho.

Reply:
yes, fixed to get the linter passing 🤷‍♂️

Reply:
In my experience you can change it back and still have a good probability that it works :) (I think it's independent of the space)

Reply:
done